jbpm-wb-forms/jbpm-wb-forms-client/src/test/java/org/jbpm/workbench/forms/client/display/displayers/pr/AbstractStartProcessFormDisplayerTest.java

```diff
@@ -53,13 +52,16 @@
     @Before
     public void setupMocks() {
         processServiceCaller = new CallerMock<ProcessService>(processService);
-        getStartProcessFormDisplayer().setProcessService(processServiceCaller);
     }
 
     @Test
     public void testNotificationOnStartProcess() {
+        getStartProcessFormDisplayer().setNewProcessInstanceEvent(newProcessInstanceEvent);
```

Comment:
these set service methods should be in the _setupMocks_ method.

Reply:
if you override _setupMocks_ in the child class you can handle that. So execute what is _init_ today and then the parent method.